### ISSUE TITLE
Python 3.8 not supported

### ISSUE THREAD

**Issue by ricardogsilva** [overflowdigital/Flask-OpenAPI#81]
**System information**
- OS version: Ubuntu 22.04
- Flask-OpenAPI version: 2.1.1
- Flask version: 3.0.0
- Python version: 3.8.16
- OpenAPI Spec version (if applicable):

**Describe the current behavior**
This seems like a nice and very useful project. I was trying it out for the first time in order to get a feel for it, but unfortunately hit some version mismatch:

After I decorate a view using the `@openapi_spec` decorator, as instructed in the docs, when launching the flask dev server I am faced with the following error:

```python
File "/home/ricardo/dev/pygeoapi/.venv38/lib/python3.8/site-packages/flask_openapi/__init__.py", line 8, in <module>
    from flask_openapi.core.decorators import openapi_spec, swag_from  # noqa: F401
  File "/home/ricardo/dev/pygeoapi/.venv38/lib/python3.8/site-packages/flask_openapi/core/decorators.py", line 5, in <module>
    from flask_openapi.core.marshmallow_apispec import Schema
  File "/home/ricardo/dev/pygeoapi/.venv38/lib/python3.8/site-packages/flask_openapi/core/marshmallow_apispec.py", line 7, in <module>
    from flask_openapi.core.validation import validate
  File "/home/ricardo/dev/pygeoapi/.venv38/lib/python3.8/site-packages/flask_openapi/core/validation.py", line 10, in <module>
    from flask_openapi.core.parser import parse_definitions, parse_schema
  File "/home/ricardo/dev/pygeoapi/.venv38/lib/python3.8/site-packages/flask_openapi/core/parser.py", line 10, in <module>
    from flask_openapi.utils.paths import get_path_from_doc, get_root_path
  File "/home/ricardo/dev/pygeoapi/.venv38/lib/python3.8/site-packages/flask_openapi/utils/paths.py", line 36, in <module>
    def get_path_from_doc(full_doc: str) -> tuple[str, Literal["yml", "yaml"]]:
TypeError: 'type' object is not subscriptable
```

The above message seems to imply that, despite claiming support for Python 3.8, this project does not really support it (hence why the issue being categorized as a bug - I hope you will fix it, rather than drop Py38 :cold_sweat: ). The error seems to boil down to this:

```python
tuple[str, Literal["yml, "yaml"]]
```

On Python < 3.9 it is not possible to use `tuple[]`, this only got introduced in [PEP 585](https://peps.python.org/pep-0585/), which debuts on Python 3.9

**Describe the expected behavior**
I would expect the code example code documented on the project's docs to work flawlessly on Python 3.8, as the project claims to support it.

**Standalone code to reproduce the issue**
Create a virtualenv with Python 3.8 and try to replicate the steps mentioned in your [Getting Started](https://github.com/overflowdigital/Flask-OpenAPI/wiki#getting-started) docs

**Comment by joshuathompsonlindley** [comment]
Apologies for the issue, I'll admit this was an accident on our part when typing the codebase. We will restore the 3.8 compatible type annotations and get a new release ASAP.

**Comment by joshuathompsonlindley** [comment]
This was fixed in release 2.2.0. Thanks for reporting.
